### PR TITLE
Added Fade-In CSS animation

### PR DESCRIPTION
**Pull request by SteveAndreou**
Added simple fade-in css animation for new list items.

Made a few additional css changes to address the following:
- new fadeIn animation was conflicting with highlighting, added important to highlighting tag
- removed user selection from list items (stop click and drag highlighting on UI)

These changes are made to address issue #9

### CODE REVIEW

File: src/TaskList.svelte [hunk 1]
```diff
@@ -78,14 +78,18 @@ export let scrollingDuration = 2000;
 
   .entry {
     display: flex;
+    animation-name: fadeIn;
```
Comment (negue) [inline src/TaskList.svelte]: you could apply those values if `:not(.entry-done):not(.entry-highlight)` that way you don't need to apply `!important`

**Review by negue — CHANGES_REQUESTED** [review]
Only that one thing, otherwise thats perfect :)

Thank you!

**Comment by SteveAndreou** [comment]
@negue thanks for the feedback, I've implemented your suggested changes.

**Comment by negue** [comment]
Thank you for your first contribution!!

If you want and have an idea, feel free to add a "item remove" animation as well :)

@all-contributors please add @SteveAndreou for code

**Comment by negue** [comment]
@all-contributors please add @SteveAndreou for code

**Pull request merged**